scripts/run-update-lambda

```diff
@@ -26,4 +26,4 @@ docker run --rm \
   --memory-swap="${MEM}" \
   --cpus="${CPUS}" \
   --name="${NAME}" \
-  lambci/lambda:python3.7 update.lambda_handler
+  lambci/lambda:python3.9 update.lambda_handler
```

Comment:
Not suggesting it should be changed here, but this is a very good reason to do a wider refactor of the AV scanner - this image hasn't been updated in 2 years.

Actually, having written that, I don't see a 3.9 version of that image at all
```
$ docker image pull lambci/lambda:python3.9
Error response from daemon: manifest for lambci/lambda:python3.9 not found: manifest unknown: manifest unknown
```